packages/manager/apps/pci-instances/src/pages/instances/instance/dashboard/components/InstanceGeneralInfoBlock.component.tsx

```diff
@@ -160,34 +148,32 @@ const InstanceGeneralInfoBlock: FC = () => {
         </div>
       </DashboardTileBlock>
       {instance && (
-        <TileBlock>
+        <DashboardTileBlock isLoading={false}>
```

Comment:
We could prevent from using unnecessary `isLoading` property :)

Reply:
done